components/layout.tsx

```diff
@@ -1,4 +1,6 @@
-import { makeStyles, createStyles, Typography } from '@material-ui/core';
+import { Typography } from '@mui/material';
+import makeStyles from '@mui/styles/makeStyles';
```

Comment:
Could we combine these from `@mui/styles`?
```suggestion
import { makeStyles, createStyles  } from '@mui/styles;
```